python/importer/main.py

```diff
@@ -12,6 +12,7 @@
 from common.utils import get_conn_params
 import common.constants as constants
 from .run_analysis import main as run_analysis
+from .remove_old_data import main as remove_old_data
```

Comment:
`import main as remove_old_data` alias style might bring more confusion than convenience in the long run, and it's probably not a common way to do it in Python. Of course this follows the same pattern we already used above with `run_analysis`, but this pattern could still be worth checking.

I'd rather just rename the current `main()` function in `remove_old_data.py` into `remove_old_data()`, and import it here as `from .remove_old_data import remove_old_data`. And, for now, keep `remove_old_data.py` file as a module only, not executable, by removing the `if __name__ == '__main__': main()` part. If we later need to make that file executable, then the pythonic way to do it would be like this, I think:

```python
# imports and def remove_old_data() here...
def main():
    remove_old_data()

if __name__ == '__main__':
    main()
```

This looks quite boilerplate, but it tells explicitly what the py file does as executable, and separates executable logic from functions that can be imported elsewhere.